Brewfile

```diff
@@ -1,2 +1,3 @@
 brew 'postgresql'
 brew 'node'
+brew "nginx"
```

Comment:
@atareshawty you need to add `brew tap github/bootstrap` to the top of this file see: https://github.com/github/homebrew-bootstrap#usage